pkg/gpio/gpio_linux.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package gpio provides functions for interacting with GPIO pins via the
+// GPIO Sysfs Interface for Userspace.
+package gpio
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+const gpioPath = "/sys/class/gpio"
+
+// Value represents the value of a gpio pin
+type Value bool
+
+// Gpio pin values can either be low (0) or high (1)
+const (
+	Low  Value = false
+	High Value = true
+)
+
+func (v Value) Dir() string {
+	if v == Low {
+		return "low"
+	}
+	return "high"
+}
+
+func (v Value) String() string {
+	if v == Low {
+		return "0"
+	}
+	return "1"
+}
+
+// SetGpioOutputValue configures the gpio as an output pin with the given value.
+func SetGpioOutputValue(pin int, val Value) error {
```

Comment:
Remove Gpio from all the names, please.

Otherwise, it reads like this when used in other packages

gpio.SetGpioOutputValue

But

gpio.SetOutputValue is sufficient

Reply:
In Go-speak, they call this stuttering - gpio.GpioValue would stutter

Reply:
Done!